Quantum-Control-Applications/Superconducting/Two-Flux-Tunable-Transmons/Standard Configuration/configuration_with_lf_fem_and_octave.py

```diff
@@ -0,0 +1,759 @@
+ge_threshold_q2 = 0.0
+
+#############################################
+#                  Config                   #
+#############################################
+config = {
+    "version": 1,
+    "controllers": {
+        con: {
+            "type": "opx1000",
+            "fems": {
+                fem: {
+                    "type": "LF",
+                    "analog_outputs": {
+                        # Resonator I-quadrature
+                        1: {
+                            "offset": 0.0,
+                            # The "output_mode" can be used to tailor the max voltage and frequency bandwidth, i.e.,
+                            #   "direct":    1Vpp (-0.5V to 0.5V), 750MHz bandwidth (default)
+                            #   "amplified": 5Vpp (-2.5V to 2.5V), 330MHz bandwidth
+                            "output_mode": "direct",
+                            # The "sampling_rate" can be adjusted by using more FEM cores, i.e.,
+                            #   1 GS/s: uses one core per output (default)
+                            #   2 GS/s: uses two cores per output
+                            # NOTE: duration parameterization of arb. waveforms, sticky elements and chirping
+                            #       aren't yet supported in 2 GS/s.
+                            "sampling_rate": sampling_rate,
+                            # At 1 GS/s, use the "upsampling_mode" to optimize output for
+                            #   modulated pulses (optimized for modulated pulses):      "mw"    (default)
+                            #   unmodulated pulses (optimized for clean step response): "pulse"
+                            "upsampling_mode": "mw",
+                        },
+                        # Resonator Q-quadrature
+                        2: {
+                            "offset": 0.0,
+                            "output_mode": "direct",
+                            "sampling_rate": sampling_rate,
+                            "upsampling_mode": "mw",
+                        },
+                        # Qubit 1 XY I-quadrature
+                        3: {
+                            "offset": 0.0,
+                            "output_mode": "direct",
+                            "sampling_rate": sampling_rate,
+                            "upsampling_mode": "mw",
+                        },
+                        # Qubit 1 XY Q-quadrature
+                        4: {
+                            "offset": 0.0,
+                            "output_mode": "direct",
+                            "sampling_rate": sampling_rate,
+                            "upsampling_mode": "mw",
+                        },
+                        # Qubit 1 XY I-quadrature
+                        5: {
+                            "offset": 0.0,
+                            "output_mode": "direct",
+                            "sampling_rate": sampling_rate,
+                            "upsampling_mode": "mw",
```

Comment:
This should be Q1 flux line, with amplified and pulse I guess

Reply:
It should be Q2 XY- when I copy-pasted I just forgot to change from qubit 1 to 2.